core-web/libs/template-builder/src/lib/directives/container-options.directive.ts

```diff
@@ -0,0 +1,80 @@
+import { Observable, Subscription } from 'rxjs';
+
+import { Directive, OnDestroy, OnInit, Optional, Self } from '@angular/core';
+
+import { Dropdown } from 'primeng/dropdown';
+
+import { debounceTime, map, switchMap, tap } from 'rxjs/operators';
+
+import { DotContainersService } from '@dotcms/data-access';
+import { DotContainerEntity, DotDropdownSelectOption } from '@dotcms/dotcms-models';
+
+const DEFAULT_LABEL_NAME_INDEX = 'label';
+const DEFAULT_VALUE_NAME_INDEX = 'value';
+
+/**
+ * Directive to set a default configuration of Dropdown or MultiSelect (PrimeNG) and translate the label of the options
```

Comment:
`or MultiSelect (PrimeNG) and translate the label of the options` I think not implemented.

Reply:
Yeah, we don't need multiselect yet.